tests/openadapt/test_recording.py

```
@@ -3,15 +3,133 @@
+            except ChildProcessError:
+                pass
+        except OSError:
+            pass  # Process is gone
+    except Exception as e:
+        logger.warning(f"Error while terminating process: {e}")
+
+@pytest.fixture
+def setup_ci_mocks(monkeypatch):
+    """Setup mocks for CI environment testing"""
+    monkeypatch.setenv('CI', 'true')
+    monkeypatch.setattr(Quartz, 'CGWindowListCopyWindowInfo', mock_window_list)
```

Ruff

Undefined name `Quartz`

(F821)